content/en/docs/collector/getting-started.md

```diff
@@ -162,7 +162,7 @@ Every Collector release includes EXE and MSI packaging for Windows amd64 systems
 The MSI packaging includes a default configuration that can be found at
 `\Program Files\OpenTelemetry Collector\config.yaml`.
 
-> Please note the Collector service is not automatically started
+> Please note the Collector service is automatically started
```

Comment:
I'll let @austinlparker, @tedsuo et al. comment on this, but if the service is now automatically started, I'd consider just dropping the note altogether. My 2 cents.

Reply:
Might as well err on the side of being overly-explicit?